oc_ftp_upload_worker/upload_worker.py

```
@@ -2,25 +2,82 @@
+        """
+        logging.debug('Reached UploadWorkerApplication.custom_connect')
+        self.pgq = PgQAPI.PgQAPI()
+        logging.debug('self.pgq: [%s]' % self.pgq)
+
+    def custom_run(self):
+        """
+        Alternative run method that uses db instead of amqp
+        """
+        logging.debug('Reached UploadWorkerApplication.custom_run')
+        msg = None
+        err_message = None
```

Ruff (0.13.2)

Local variable `err_message` is assigned to but never used

Remove assignment to unused variable `err_message`

(F841)